embedded-aerospike/src/main/java/com/playtika/test/aerospike/EmbeddedAerospikeBootstrapConfiguration.java

```diff
@@ -99,6 +105,7 @@ private void registerAerospikeEnvironment(GenericContainer aerospike,
         map.put("embedded.aerospike.host", host);
         map.put("embedded.aerospike.port", mappedPort);
         map.put("embedded.aerospike.namespace", properties.namespace);
+        map.put("embedded.aerospike.featureKey", properties.featureKey);
```

Comment:
we do not need to expose this variable, since we are setting it manually

Reply:
Removed